client/src/Views/Landing/ProcessGmailToken.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Google authorisation process redirects here. Checks for success.
+ * Sends authorisation code to back-end
+ */
+
+import React, { Component } from "react";
+import {
+  CircularProgress,
+  DialogTitle,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Button,
+  withStyles,
+} from "@material-ui/core";
+import { withRouter } from "react-router-dom";
+import { postCode } from "Utils/api";
+
+const styles = {
+  // center the spinner
+  spinner: {
+    display: "block",
+    margin: "auto",
+  },
+};
+
+class ProcessToken extends Component {
```

Comment:
can we change this to something like GmailAuthResultDialog - something that is a bit more readable from the perspective of the user... it took me a while to figure out what this did

Reply:
Good point. Fixed.